src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -156,6 +156,9 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						if (!value.isEmpty() && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H264)) {
 							media.setAvcLevel(getAvcLevel(value));
 						}
+						if (isNotBlank(value) && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H265)) {
```

Comment:
I'd say that storing the "raw" value from ```Format_Profile``` in ```DLNAMediaInfo``` and the database would be better. The parsing of the value should be done in ```DLNAMediaInfo.getAvcLevel()``` and ```DLNAMediaInfo.getHevcLevel()``` instead. Parsing this will be so quick that doing it each time won't be noticable, and the benefit is that every ```DLNAMediaInfo``` instance kept in memory (which can be very many) and each database record will be smaller. It will also be enough with one set of locks.

This is where I feel like showing is easier than explaining, but I won't do that this time...